t/03_tools.t

```diff
@@ -2,14 +2,16 @@ use strict;
 use warnings;
 use Test::More tests => 3;
 use Test::Alien;
+use Alien::Build;
 use Alien::libtiff;
 use File::Temp;
 
 alien_ok 'Alien::libtiff';
+my $build = Alien::Build->load('./alienfile');
 
 SKIP: {
     skip "The default system install on Windows doesn't include libtiff tools",
-      2 unless $ENV{ALIEN_INSTALL_TYPE} eq 'share';
+      2 unless $build->install_type eq 'share';
```

Comment:
This can be changed to the more direct:

```perl
unless Alien::libtiff->install_type('share');
```

Apologies for copy-paste issue with previous comment that meant it was not very clear.